parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -511,15 +511,19 @@ finally
 {
     # Generate CSE result so it can be returned as the CSE response in csecmd.ps1
     $ExecutionDuration=$(New-Timespan -Start $StartTime -End $(Get-Date))
-    Write-Log "CSE ExecutionDuration: $ExecutionDuration"
-
-    # Windows CSE does not return any error message so we cannot generate below content as the response
-    # $JsonString = "ExitCode: `"{0}`", Output: `"{1}`", Error: `"{2}`", ExecDuration: `"{3}`"" -f $global:ExitCode, "", $global:ErrorMessage, $ExecutionDuration.TotalSeconds
-    Write-Log "Generate CSE result to $CSEResultFilePath : $global:ExitCode"
-    echo $global:ExitCode | Out-File -FilePath $CSEResultFilePath -Encoding utf8

```

Comment:
I think we can keep this line `echo $global:ExitCode | Out-File -FilePath $CSEResultFilePath -Encoding utf8` because all Windows nodes have the file `$CSEResultFilePath`. Otherwise, we can remove the file if it's useless?

Reply:
I tried it but unfortunately it will not return the detailed error message in the cse response. We only can use `exit $ExitCode` or `throw 'error code with the message'`.